.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
+  "parser": "babel-eslint",
+  "extends": "airbnb",
+  "globals": {
+    "__DEV__": true
+  },
+  "env": {
+    "browser": true,
+    "mocha": true
+  },
+  "rules": {
+    "no-confusing-arrow": 0,
+    "react/jsx-quotes": 0,
```

Comment:
`` react/jsx-quotes `` is deprecated according to https://github.com/airbnb/javascript/issues/513. Remove this maybe?